lib/pdftotext/cli.rb

```diff
@@ -8,7 +8,7 @@ class CLI
     def run_command(*args)
       options = DEFAULT_OPTIONS.merge(args.pop)
       args = args.concat options_to_args(options)
-      output, status = Open3.capture2e(bin_path, *args)
+      output, status = Open3.capture2e("#{bin_path} #{args.join(" ")}")
```

Comment:
Okay, digging deeper the issue was the way command line flags with arguments were being passed. Rolled-back this change, added a test case, and made a new fix.